ql/src/test/queries/clientpositive/explain_systest_password.q

```diff
@@ -0,0 +1,12 @@
+--! qt:sysdb
+
+select 1
+from sys.TBLS t
+    join sys.DBS d on t.DB_ID = d.DB_ID
+limit 1;
+
+explain extended
```

Comment:
Could we use a more simpler qtest instead?

Reply:
What would you prefer by a simpler qtest? 

It only selects a constant. And the output is limited to one row.

Reply:
ok, it is reasonable for the `explain` case. 

If we also need to fix the "show create table" use case, then only a simple SQL command "show create table" is enough.

Reply:
@InvisibleProgrammer, why does it have to be a sys db?